src/components/Widget/Widget.js

```diff
@@ -47,7 +47,7 @@ const Widget = ({
 }) => {
   const ResolvedImage = () =>
     imageSrc ? (
-      <Image imageSrc={imageSrc} fit="cover" />
+      <Image imageSrc={imageSrc} fit="contain" />
     ) : (
       <Image fit="cover">{image}</Image>
```

Comment:
@alexandra-stepanova And here?

Reply:
@varya I have changed it as well. Thank you.